ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -129,7 +129,9 @@ def cleanup(cluster_name, cluster_id, upi=False, failed_deletions=None):
     delete_cluster_buckets(cluster_name)
 
 
-def get_clusters(time_to_delete, region_name, prefixes_hours_to_spare):
+def get_clusters(
+    time_to_delete, region_name, prefixes_hours_to_spare, cluster_pattern=None
```

Comment:
cluster_pattern is not documented in docstring

Reply:
done